app/models/tag.rb

```diff
@@ -1,3 +1,2 @@
 class Tag < ActiveRecord::Base
-	
```

Comment:
Did you mean to delete what was here? Now there's an empty class

Reply:
Yes